### PR TITLE
🧑‍💻(frontend) fix storybook

### PR DESCRIPTION
**Pull request by rlecellier**
Since cunnigham is used, most of our stories were broken because CunninghamProvider was missing.

### CODE REVIEW

File: src/frontend/.storybook/preview-body.html [hunk 1]
```diff
@@ -1 +1 @@
-<div id="modal-exclude" style="min-height: 0;"></div>
\ No newline at end of file
+<div id="modal-exclude" style="height: auto;"></div>
```
Comment (jbpenrath) [inline src/frontend/.storybook/preview-body.html:1]: A blank line is missing

File: src/frontend/.storybook/preview.tsx [hunk 2]
```diff
@@ -34,7 +35,9 @@ export const decorators = [
   (Story) => (
     <IntlProvider locale="en">
       <IconsWrapper>
-        <Story />
+        <CunninghamProvider>
+          <Story />
+        </CunninghamProvider>
```
Comment (jbpenrath) [inline src/frontend/.storybook/preview.tsx:40]: Did you check that some stories does not import CunninghamProvider internally ?

**Review by jbpenrath — APPROVED** [review]
LGTM ✅

File: src/frontend/.storybook/preview.tsx (hunk 2 above)
Comment (rlecellier) [reply src/frontend/.storybook/preview.tsx:40]: i hadn't, it's done now some stories needed to have the provider removed.

**Pull request merged**